android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -466,6 +471,47 @@ class TranslationsMiddleware(
         )
     }
 
+    /**
+     * Fetches the expected translation model download size assuming the user intends to complete
+     * a translation using detected defaults.
+     *
+     * If detected defaults are available, then this will sets the values on
+     * [SessionState.translationsState] on the [BrowserStore]
+     * via [TranslationsAction.FetchTranslationDownloadSizeAction].
```

Comment:
I am feeling that this piece of the comment is a bit forced? (possibility because it is copying similar comments from above) It's kinda repeating what the code is doing which from my experience usually means the comment is probably not adding much. The comment is also a bit vague about what values it is setting, which makes the comment itself feels somewhat unsatisfying in terms of being clear what it is doing. In reality, `FetchTranslationDownloadSizeAction` will attempt to fetch the download size and try and populate `translationDownloadSize`.

Reply:
Yeah, agreed! Rephrased it